shared/ff_util/include/ff_util/ff_action.h

```diff
@@ -214,6 +216,8 @@ class FreeFlyerActionClient {
       &FreeFlyerActionClient::DeadlineTimeoutCallback, this, true, false);
     timer_poll_ = nh->createTimer(to_poll_,
         &FreeFlyerActionClient::ConnectPollCallback, this, false, false);
+    timer_response_delay_ = nh->createTimer(to_response_delay_,
+        &FreeFlyerActionClient::ResultDelayCallback, this, false, false);
```

Comment:
If you make this a one shot timer, you don't have to worry about stopping the timer in the result delay callback.

Reply:
done :)

Reply:
I also tested it and it still works as expected